src/core/Akka.Tests/Serialization/SerializationSpec.cs

```diff
@@ -453,7 +453,7 @@ public void Can_serialize_ActorPathContainer()
         [Fact]
         public void Can_serialize_singleton_messages()
         {
-            var message = Terminate.Instance;
+            var message = new Terminate();
```

Comment:
**NEED TO FIX**: should change this to `PoisonPill`; otherwise the test is invalid.